aim/db/infra_model.py

```diff
@@ -131,9 +131,11 @@ class HostDomainMapping(model_base.Base, model_base.AttributeMixin):
     __tablename__ = 'aim_host_domain_mapping'
 
     host_name = sa.Column(sa.String(128), primary_key=True)
-
-    vmm_domain_name = sa.Column(sa.String(64))
-    physical_domain_name = sa.Column(sa.String(64))
+    domain_name = sa.Column(sa.String(64), primary_key=True, nullable=True)
```

Comment:
it doesn't make sense to me that domain_name can be Null.....

Reply:
This was left-over from some other attempt at the patch -- will remove.